_posts/2020-02-10-q1-2020-update.md

```diff
@@ -0,0 +1,186 @@
+en-only: true
+---
+
+<div class='responsive-youtube-container'>
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/oImSdIAsywA" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+
+The year 2019 was, by almost any reasonable measure, an incredibly successful year for the Bisq network. 
+
+Here is a sampling of the milestones achieved:
+
+- v1.0 launched after 4+ years of work
+- Bisq DAO launched as perhaps the first and only bitcoin-based DAO
+    - Of 20 100 total trades done since the DAO launch in 2019, 8 589 used BSQ to pay trading fees
+- New trading records reached
+    - USD-adjusted volume up 2.6x (from 5 256 BTC to 13 782 BTC)
+- New trade protocol deployed 
+    - Trades are now secured with 2-of-2 multisig escrows instead of 2-of-3 multisig escrows
+- Account signing
+    - Novel mechanism to increase counterparty confidence for fiat trades while maintaining privacy
+
+If Bisq just set out to be a successful software project that facilitated bitcoin trading, this would be it. There would be no need to step back and reevaluate anything. Just keep developing, keep growing, and let things evolve as they do.
+
+But the vision for Bisq is a bit grander.
+
+The vision is for Bisq to become _the exchange layer for the Bitcoin protocol stack_, organized as a thoroughly decentralized DAO that sovereignly manages and funds itself. This requires the project to manage itself in a way different from your typical open software project, with a greater focus on efficiency and effectiveness.
+
+As a result, some challenges emerged as a result of Bisq's 2019 success that hampered the project's ability to continue to effectively achieve its vision:
+
+- Lack of management led to suboptimal resource allocation
+    - Overspending on low priorities and underspending on high priorities
+- Contributor base grew, but developer capacity remained limited
+- Volatile volumes resulted in revenues insufficient to support more developers
+- Weak support often left users frustrated, hampering growth and development efforts
+
+It's become clear that while the Bisq DAO provides a solid technical basis for a decentralized operation, the human side of the Bisq DAO wasn't as solid.
+
+## A New Structure
+
+To that end, Bisq is reorganizing its operations into **5 teams** with **goals** and **budgets**.
+
+Until now, Bisq's contributor network didn't really have any structure. Aside from designating people for roles to handle ongoing responsibilities, the project has largely functioned as an amorphous group of people who contributed to the project on their own accord—in other words, a free-for-all—with no real management, no formal constraints, and no analysis. This worked well while the contributor group was small, since priorities aligned nicely with the people who were around to take them on.
```

Comment:
`s/didn't really have any structure/didn't have much structure/`?

Reply:
`s/amorphous group of people/amorphous group of individuals/` - avoids repeating the word people in the same sentence, flows better.